src/test/java/in/org/projecteka/hiu/dataflow/TestBuilders.java

```diff
@@ -30,6 +36,15 @@
         return easyRandom.nextObject(DataFlowRequestResult.DataFlowRequestResultBuilder.class);
     }
 
+    public static HealthInformationFetchRequest.HealthInformationFetchRequestBuilder healthInformationRequest() {
+        return easyRandom.nextObject(HealthInformationFetchRequest.HealthInformationFetchRequestBuilder.class);
+    }
+
+    public static List<DataPartDetail> dataPartDetails(int size, String requester, HealthInfoStatus status) {
+        var builders = easyRandom.objects(DataPartDetail.DataPartDetailBuilder.class, size);
+        return builders.map(builder -> builder.requester(requester).status(status).build()).collect(Collectors.toList());
+    }
```

Comment:
NIC: Suggestion would be to return only builders.

Reply:
Yes sure.

Reply:
done